tests/e2e.py

```diff
@@ -4,43 +4,31 @@
 from selenium.webdriver.common.keys import Keys
 from trello import TrelloClient
```

Comment:
We can now delete this import - and the underlying file since we're not using it any more. We can always recover it with git later